### ISSUE TITLE
Rename to fabric-template-mod

### ISSUE THREAD

**Issue by haykam821** [FabricMC/fabric-example-mod#79]
This isn't really an example mod, and could confuse users. If it's an example mod, it should be adding actual content rather than printing a line on initialization.

**Comment by Prospector** [comment]
`fabric-mod-template` sounds a bit more natural to me

**Comment by Technici4n** [comment]
![bikeshed](https://user-images.githubusercontent.com/13494793/106797966-9b64d380-665d-11eb-86d6-15230f6f2722.png)

**Comment by actuallyasmartname** [comment]
I agree with this hehe

**Comment by Jefaxe** [comment]
Yeah, I was expecting a small mod that adds a block and item so you could learn from it. `fabric-mod-template` sounds better for what this is.

**Comment by antonilol** [comment]
or add a block and an item so it is an example mod :)

**Comment by Gamer069** [comment]
this isn't really an issue, but yeah, it sounds better, i agree
